modules/appeals_api/spec/factories/supplemental_claims.rb

```diff
@@ -53,14 +53,13 @@
   factory :extra_supplemental_claim_v0,
           class: 'AppealsApi::SupplementalClaim', parent: :extra_supplemental_claim do
     api_version { 'V0' }
-    auth_headers { FixtureHelpers.fixture_as_json 'supplemental_claims/v0/valid_200995_headers_extra.json' }
+    auth_headers { FixtureHelpers.fixture_as_json 'supplemental_claims/v0/valid_200995_headers.json' }
```

Comment:
IMO it no longer makes sense to keep `headers_extra` or `headers_minimum` files for supplemental claims, because the only header fields available (consumer username and ID, which are optional) are covered by the default headers file